TTD/LiveRecorderApiSample/main.cpp

```diff
@@ -0,0 +1,190 @@
+#include <Windows.h>
```

Comment:
good sample: not trivial, not complicated.

Reply:
Thanx!